src/element.ts

```diff
@@ -97,20 +86,38 @@ export abstract class Element {
   protected registry?: Registry;
 
   /**
-   * Some elements include text.
    * The `textFont` property contains information required to style the text (i.e., font family, size, weight, and style).
-   * It is undefined by default, and can be set using `setFont(...)` or `resetFont()`.
    */
-  protected textFont?: Required<FontInfo>;
+  protected textFont: Required<FontInfo>;
+  protected text = '';
```

Comment:
Element.text seems like the new important addition. Can we add a good comment for this property? Some of your description of this PR would be useful here. Basically, every Element that is placed on the staff is fundamentally a `text: string` styled by a SMuFL font.

Reply:
done :)